frontend/src/static/css/v_summary.scss

```diff
@@ -9,6 +9,8 @@
   .icon-button {
     color: mui-color('grey');
     cursor: pointer;
+    padding-left: 1.2px;
+    padding-right: 1.2px;
```

Comment:
why introducing this additional padding?

Reply:
This is to add additional padding for the background of the selected icon since I have set the background color to be green. The coloured background may be bounded too tightly by the border without this padding, which can make the icon look a bit strange. If there is any alternative design of the highlight effect, then the padding may not be needed.

Reply:
@damithc Hello professor, can I ask for your opinion on the current design of the highlight effect? Many thanks

Reply:
Sorry, I didn't think about this earlier. The current one feels too subtle. It defeats the purpose if the highlighted item is not easy to spot. What about doing something to the entire box rather than just the item only e.g., make the background of the entire area light yellow?

![image](https://user-images.githubusercontent.com/1673303/108988045-f7f06700-76ce-11eb-928d-efda54c9058c.png)

Reply:
Thanks for the suggestion. However, in order to distinguish whether it is the authorship contribution or the commit panel that is currently being opened, will it be more suitable to highlight the icon with a more obvious visual effect instead?

Reply:
Well, in the unlikely case the user forgot which one, he only has to look to the right panel to see which of the two it is, yes? :-)
In any case, no harm doing both i.e., colorize the icon and well as the full box.

Reply:
Sure, no problem